pkg/planner/core/casetest/dag/testdata/plan_suite_out.json

```diff
@@ -16,7 +16,7 @@
       },
       {
         "SQL": "select * from t where (t.c > 0 and t.c < 2) or (t.c > 4 and t.c < 6) or (t.c > 8 and t.c < 10) or (t.c > 12 and t.c < 14) or (t.c > 16 and t.c < 18)",
-        "Best": "TableReader(Table(t)->Sel([or(or(and(gt(test.t.c, 0), lt(test.t.c, 2)), and(gt(test.t.c, 4), lt(test.t.c, 6))), or(and(gt(test.t.c, 8), lt(test.t.c, 10)), or(and(gt(test.t.c, 12), lt(test.t.c, 14)), and(gt(test.t.c, 16), lt(test.t.c, 18)))))]))"
+        "Best": "IndexLookUp(Index(t.c_d_e)[[1,1] [5,5] [9,9] [13,13] [17,17]], Table(t))"
```

Comment:
Why is this changed? The old and the new execution plans both don't contain `IndexMerge`.

Reply:
i'm quite strange about this, after debug inside i find some difference in the estimation of selectivity
<img width="513" alt="image" src="https://github.com/pingcap/tidb/assets/15825830/d0086489-2746-4196-a2d9-f852311dbe4b">
<img width="394" alt="image" src="https://github.com/pingcap/tidb/assets/15825830/dac7b469-5e12-4156-9850-694196915b62">
and finally leading the ds.statsCount difference about 1250 vs 50